### ISSUE TITLE
"5/8" gets treated as IP

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#2132]
When doing `:open 5/8`:

```
10:42:21 DEBUG    commands   command:run:538 Calling qutebrowser.browser.commands.CommandDispatcher.openurl(<qutebrowser.browser.commands.CommandDispatcher>, '5/8', False, False, False, False, None)
10:42:21 DEBUG    url        urlutils:get_path_if_valid:359 Checking if '5/8' is a path
10:42:21 DEBUG    url        urlutils:is_url:244 Checking if '5/8' is a URL (autosearch=naive).
10:42:21 DEBUG    url        urlutils:is_url:281 Checking via naive check
10:42:21 DEBUG    url        urlutils:is_url:285 url = True
10:42:21 DEBUG    url        urlutils:fuzzy_url:192 URL is a fuzzy address
10:42:21 DEBUG    url        urlutils:fuzzy_url:195 Converting fuzzy term '5/8' to URL -> http://0.0.0.5/8
```

**Comment by tuxlovesyou** [comment]
Is this supposed to be a DDG search?

**Comment by The-Compiler** [comment]
Yep, that's what I would expect.

Here the situation looks like this from a quick look, in `utils.urlutils` in `_is_url_naive:

- `ipaddress.ip_address` doesn't treat it as URL
- The `QHostAddress` check does not fail
- `QUrl::fromUserInput` returns an URL for `0.0.0.5/8` which then returns `True` because it contains a dot.

**Comment by The-Compiler** [comment]
Fixed via #5056
